components/cluster/pacemaker/Makefile

```
@@ -53,18 +52,18 @@ CONFIGURE_OPTIONS += --sysconfdir=/etc
 CONFIGURE_OPTIONS += --sharedstatedir=/var/hacluster
 CONFIGURE_OPTIONS += --localstatedir=/var
 CONFIGURE_OPTIONS += --with-corosync
-CONFIGURE_OPTIONS += --with-heartbeat=no
-CONFIGURE_OPTIONS += --with-snmp
-CONFIGURE_OPTIONS += --with-esmtp
-CONFIGURE_OPTIONS += --with-cs-quorum
 CONFIGURE_OPTIONS += --with-acl=no
 CONFIGURE_OPTIONS += --enable-upstart=no
 CONFIGURE_OPTIONS += --enable-systemd=no
 CONFIGURE_OPTIONS += --enable-fatal-warnings=no
 CONFIGURE_OPTIONS += --disable-rpath
+CONFIGURE_OPTIONS += --with-logdir=/var/log/hacluster
+CONFIGURE_OPTIONS += --with-bundledir=/var/log/hacluster/bundles
+CONFIGURE_OPTIONS += --with-openssl=yes
 CONFIGURE_OPTIONS += $(CONFIGURE_OPTIONS.$(BITS))
 
 CONFIGURE_ENV.64 += LT_SYS_LIBRARY_PATH="/lib/$(MACH64):/usr/lib/$(MACH64)"
+CONFIGURE_ENV.64 += PYTHON=/usr/bin/amd64/python2.7
```

Will it work with Python 3.5? If so then set `PYTHON_VERSION=3.5` and then:
```suggestion
CONFIGURE_ENV.64 += PYTHON=$(PYTHON)
```

---

Following https://clusterlabs.org/pacemaker/doc/en-
US/Pacemaker/1.1/html/Pacemaker_Development/_python_compatibility.html
it should work with Python 3.5.
I will change this PR and pcs and crmsh (if possible).